fsleyes_plugin_shimming_toolbox/st_plugin.py

```diff
@@ -728,6 +729,280 @@ def create_sizer_run(self):
+                "name": "vop",
+                "button_function": "select_file",
+                "info_text": "Siemens SarDataUser.mat file (located in C:/Medcom/MriProduct/Physconfig)"
+            },
+            {
+                "button_label": "SAR factor",
+                "name": "sar_factor",
+                "default_text": "1.5",
+                "info_text": """Factor (=> 1) to which the shimmed max local SAR can exceed the phase-only shimming max 
+                       local SAR. Values between 1 and 1.5 should work with Siemens scanners. High factors allow more 
+                       shimming liberty but are more likely to result in SAR excess on the scanner.
+                       """
+            },
+            {
+                "button_label": "Output Folder",
+                "button_function": "select_folder",
+                "default_text": path_output,
+                "name": "output",
+                "info_text": "Directory to output shim weights, B1+ maps and figures."
+            }
+        ]
+
+        component = InputComponent(self, input_text_box_metadata)
+        run_component = RunComponent(
+            panel=self,
+            list_components=[component],
+            st_function="st_b1shim --algo 2",
+            output_paths=[
+                os.path.join(path_output, "fig_resampled_riro.nii.gz"),
+                os.path.join(path_output, "fig_resampled_static.nii.gz")
+            ]
+        )
+        sizer = run_component.sizer
+        return sizer
+
+    def create_sizer_sar_eff(self, metadata=None):
+        path_output = os.path.join(CURR_DIR, "b1_shim_output")
+        input_text_box_metadata = [
+            {
+                "button_label": "Input B1+ map",
+                "name": "b1map",
+                "button_function": "select_file",
+                "info_text": "B1+ map. 4D NIfTI file as created by dcm2niix.",
+                "required": True
+            },
+            {
+                "button_label": "Input Mask",
+                "name": "mask",
+                "button_function": "select_file",
+                "info_text": "3D NIfTI file used to define the shimming region of interest"
+            },
+            {
+                "button_label": "Input VOP file",
+                "name": "vop",
+                "button_function": "select_file",
+                "info_text": "Siemens SarDataUser.mat file (located in C:/Medcom/MriProduct/Physconfig)",
+                "required": True
+            },
+            {
+                "button_label": "SAR factor",
+                "name": "sar_factor",
+                "default_text": "1.5",
+                "info_text": """Factor (=> 1) to which the shimmed max local SAR can exceed the phase-only shimming max 
+                local SAR. Values between 1 and 1.5 should work with Siemens scanners. High factors allow more shimming 
+                liberty but are more likely to result in SAR excess on the scanner.
```

Comment:
You should avoid """xxxxx""" since the help text is messed up due to the fact that in includes tabs. You can use normal "xxx" that span multiple lines.